src/k8s-configuration/azext_k8s_configuration/commands.py

```diff
@@ -5,19 +5,47 @@
 
 # pylint: disable=line-too-long
 from azure.cli.core.commands import CliCommandType
-from azext_k8s_configuration._client_factory import (cf_k8s_configuration, cf_k8s_configuration_operation)
-from ._format import k8s_configuration_show_table_format, k8s_configuration_list_table_format
+from azext_k8s_configuration._client_factory import (
+    k8s_configuration_fluxconfig_client,
+    k8s_configuration_sourcecontrol_client
+)
+from .format import (
+    fluxconfig_list_table_format,
+    fluxconfig_show_table_format,
+    fluxconfig_kustomization_list_table_format,
+    fluxconfig_kustomization_show_table_format,
+    sourcecontrol_list_table_format,
+    sourcecontrol_show_table_format
+)
 
 
 def load_command_table(self, _):
+    k8s_configuration_fluxconfig_sdk = CliCommandType(
+        operations_tmpl='azext_k8s_configuration.vendored_sdks.operations#FluxConfigurationsOperations.{}',
+        client_factory=k8s_configuration_fluxconfig_client
+    )
 
-    k8s_configuration_sdk = CliCommandType(
+    k8s_configuration_sourcecontrol_sdk = CliCommandType(
         operations_tmpl='azext_k8s_configuration.vendored_sdks.operations#SourceControlConfigurationsOperations.{}',
-        client_factory=cf_k8s_configuration)
+        client_factory=k8s_configuration_sourcecontrol_client
+    )
 
-    with self.command_group('k8s-configuration', k8s_configuration_sdk, client_factory=cf_k8s_configuration_operation) as g:
-        g.custom_command('create', 'create_k8s_configuration')
-        g.custom_command('update', 'update_k8s_configuration')
-        g.custom_command('delete', 'delete_k8s_configuration', confirmation=True)
-        g.custom_command('list', 'list_k8s_configuration', table_transformer=k8s_configuration_list_table_format)
-        g.custom_show_command('show', 'show_k8s_configuration', table_transformer=k8s_configuration_show_table_format)
+    with self.command_group('k8s-configuration flux', k8s_configuration_fluxconfig_sdk, client_factory=k8s_configuration_fluxconfig_client, is_preview=True) as g:
+        g.custom_command('create', 'flux_config_create', supports_no_wait=True)
+        g.custom_command('update', 'flux_config_update', supports_no_wait=True)
+        g.custom_command('list', "flux_config_list", table_transformer=fluxconfig_list_table_format)
+        g.custom_show_command('show', 'flux_config_show', table_transformer=fluxconfig_show_table_format)
+        g.custom_command('delete', 'flux_config_delete', supports_no_wait=True)
```

Comment:
Do we need to consider adding `confirmation=True` for the delete operation

Reply:
We are using custom delete confirmation, so we are not using the default confirmation as provided by the CLI

https://github.com/AzureArcForKubernetes/azure-cli-extensions/blob/k8s-configuration-release-1.2.0/src/k8s-configuration/azext_k8s_configuration/providers/FluxConfigurationProvider.py#L341

Reply:
Got it, thanks~